gob/k8s/README.md

```diff
@@ -257,12 +258,11 @@ in production!
 #### Canary ####
 
 For the sake of the demo, we need to generate some load on the site.
-To do so, the _reqz_ package can easily be launched as a one-off task
-with:
+To do so, [slow_cooker](https://github.com/BuoyantIO/slow_cooker) can easily
+be launched as a one-off task with:
 
 ```
-:; kubectl run --image=gobsvc/reqz:v0 reqz -- "http://$GOB_HOST/gob?limit=10&text=buoyant"
-deployment "reqz" created
+:; kubectl run --image=buoyantio/slow_cooker:1.1.0 slow-cooker -- "-qps 200 http://$GOB_HOST/gob?limit=10&text=buoyant"
```

Comment:
Yay slow_cooker! I couldn't get this to run as written. I needed the qps flag and the target to be split into two separate arguments, like so:

```
:; kubectl run --image=buoyantio/slow_cooker:1.1.0 slow-cooker -- "-qps=200" "http://$GOB_HOST/gob?limit=10&text=buoyant"
```